python/pyarrow/_plasma.pyx

```diff
@@ -451,28 +454,39 @@ cdef class PlasmaClient:
             The number of milliseconds that the get call should block before
             timing out and returning. Pass -1 if the call should block and 0
             if the call should return immediately.
-
-        Returns
+         Returns
```

Comment:
Please restore the newline and I think there is an accidental extra space in front of `Returns`